docs/description_cheat_sheet.rst

```diff
@@ -0,0 +1,91 @@
+***********************
+Description Cheat Sheet
+***********************
+
+dist_descriptions
+~~~~~~~~~~~~~~~~~
+
+dist_descriptions are used to create joint models.
+A list of dist_descriptions is passed to
+:class:`~virocon.jointmodels.GlobalHierarchicalModel`, one for each dimension.
+
+For an independent distribution:
+
+.. code:: python
+
+    # general
+    dist_description = {
+        "distribution": <distribution instance>,
+        "intervals": <interval slicer instance>,
+    }
+
+    # example
+    dist_description_v = {
+        "distribution": ExponentiatedWeibullDistribution(),
+        "intervals": WidthOfIntervalSlicer(2, min_n_points=50),
+    }
+
+For a conditional distribution:
+
+.. code:: python
+
+    # general
+    dist_description = {
+        "distribution": <distribution instance>,
+        "conditional_on": <index of distribution this dist depends on>,
+        "parameters": <dict with parameter names as keys
+                       and dependence function instances as values>,
+    }
+
+    # example
+    dist_description_hs = {
+        "distribution": ExponentiatedWeibullDistribution(f_delta=5),
+        "conditional_on": 0,
+        "parameters": {"alpha": alpha_dep, "beta": beta_dep,},
+    }
+
+
+fit_descriptions
+~~~~~~~~~~~~~~~~
+
+fit_descriptions define how to fit a model.
```

Comment:
describes "which methods are used to estimate the parameter values of a joint distribution (which fitting methods are used)"